src/IRTS/Portable.hs

```diff
@@ -5,20 +5,21 @@ Copyright   :
 License     : BSD3
 Maintainer  : The Idris Community.
 -}
+
 {-# LANGUAGE OverloadedStrings #-}
+-- XXX: Should the following be removed, moving ToJSON instance to IRTS.CodegenCommon?
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
I would prefer to not have to disable a warning here. Why does this warning happen? @melted, any comments?